### PR TITLE
update cms zh-cn lang.php file

### PR DESCRIPTION
**Pull request by wenlong-date**
When i use OctoberCMS, i found the translations of `modules/cms/lang/zh-cn/lang.php` was out of date. So, i updated it .

### CODE REVIEW

**Comment by wenlong-date** [comment]
Backend zh-cn lang.php file not finish translation at the moment . I will finish it before Jul 11  23:59:59.

**Comment by wenlong-date** [comment]
@bennothommo  I have tried my best to translate the files and finished now.

File: modules/backend/lang/zh-cn/lang.php [hunk 1]
```diff
@@ -501,7 +553,7 @@
         'label' => '媒体查找器',
         'default_prompt' => '点击 %s 按钮查找媒体项'
     ],
-   'media' => [
+    'media' => [
```
Comment (w20k) [inline modules/backend/lang/zh-cn/lang.php]: @bennothommo any ideas why CI approve space? Strange)

Comment (w20k) [reply modules/backend/lang/zh-cn/lang.php]: @wenlong-date could you remove space before 'media', doesn't seem to be correct with whitespace.

Comment (bennothommo) [reply modules/backend/lang/zh-cn/lang.php]: @w20k The extra space puts it back in correct alignment, from what I can tell, which is why it passes.

Comment (w20k) [reply modules/backend/lang/zh-cn/lang.php]: @bennothommo you're right, need a rest or two :) Thanks!

**Pull request merged**